templates/About.html

```diff
@@ -6,4 +6,121 @@
+    border-radius: 25px;
+    padding: 16px;
+    text-align: center;
+    background-color: #3A3E5E;
+    color: white
+  }
+  /*For scroll bar*/
+  body {
+    overflow-y: auto;
+  }
+/*Added from previous about_us branch*/
+  img{
+    border-radius: 48%;
+    border-collapse: separate;
+  }
+  .container {
+    float: left;
+    width: 20%;
+    justify-content: space-evenly;
+  }
+/* For responsive containers */
+  @media screen and (max-width: 600px) {
+    .container {
+      width: 100%;
+      display: block;
+      margin-bottom: 20px;
+    }
+  }
+  
+  </style>
+
+  <div class="row">
+    <div class="column">
+      <div class="card">
+        <h3>Our Story</h3>
+        <p>Our team created Eventsly because in the past, there was no easy way for you to search for and locate events in your community. We know every community has something unique and special to offer. And we know that there are exciting events happening all around you. However, in the past, learning about and locationing these events was challenging and seemingly impossible.</p>
+        <p>Existing apps would only highlight popular attractions and tourist destinations. They didn’t provide information about events happening in local communities, or how to locate them. This void has motivated us to create a simple solution making discovering and locating events in your community as easy as possible.</p>
+        <p>Our ultimate goal is to help you connect with your community, one event at a time.</p>
+      </div>
+    </div>
+  
+    <div class="column">
+      <div class="card">
+        <h3>What is Eventsly?</h3>
+        <p>Eventsly is the one-step solution to finding events in your area. To use, simply open the web application, search your location and go! Events taking place in your area will appear on the map where you can click on any that interest you to learn more. There’s no limit to the types of events you can find on Eventsly, concerts, cook-outs, get-togethers, fundraisers, and so much more!</p>
+        <p>Creating an account is fast and free. Simply navigate to the sign up option located in the upper right hand corner of each page and enter the requested information. After creating an account you will have the option to create and coordinate your own events. To create an event simply click the create an event (plus icon) at the center of the home page. If you see any upcoming events you’d like to attend you can bookmark them to locate them faster at a later date.</p>
+        <p>
+
+      </div>
+    </div> 
+ <br>
+<h1><center>Meet Our Team</center></h1>
+
+<div class="container">
+  <figure>
+    <img src="/static/images/gregory.jpg" width="200" height="200"alt="Greg"  style="box-shadow: 10px 10px 5px grey;">
+    <figcaption><center><b>Greg Flores</b></center>.</figcaption>
+  </figure>
+</div>
+
+<div class="container">
+  <figure>
+    <img src="/static/images/CJ_Fox.JPG" width="200" height="200"alt="CJ"  style="box-shadow: 10px 10px 5px grey;">
```

Comment:
In-line styling is not best practice. Should give these images a class and style them in a .css document and attach that .css file to this page.